lapis-e2e/test/nucleotideInsertions.spec.ts

```diff
@@ -51,6 +60,17 @@ describe('The /nucleotideInsertions endpoint', () => {
     expect(descendingOrderedResult.data[3]).to.have.property('insertion', 'ins_22204:CAGAA');
   });
 
+  it('advancedQuery can filter out unwanted insertions', async () => {
+    const urlParams = new URLSearchParams({
+      advancedQuery: "NOT ins_5959:TAT AND country='Switzerland'",
```

Comment:
This test is failing when I change it to this:
```suggestion
      advancedQuery: "not ins_5959:TAT AND country='Switzerland'",
```

```
{
  "error": {
    "type": "about:blank",
    "title": "Bad Request",
    "status": 400,
    "detail": "Failed to parse advanced query (line 1:14): missing NUMBER at 'A'."
  },
  "info": {
    "dataVersion": null,
    "requestId": "38fd4b38-4ffb-43b0-bda3-230859fc3d2b",
    "requestInfo": "sars_cov-2_minimal_test_config on localhost at 2025-05-14T09:35:19.105820426",
    "reportTo": "Please report to https://github.com/GenSpectrum/LAPIS/issues in case you encounter any unexpected issues. Please include the request ID and the requestInfo in your report.",
    "lapisVersion": "local",
    "siloVersion": "0.6.0"
  }
}
```

Reply:
that is to be expected - the parser requires NOT to be capitalized

Reply:
Is that what we want? Usually we are case-insensitive in LAPIS, I think we should also make the advanced queries case-insensitive.